src/raiden_libs/service_registry.py

```diff
@@ -180,5 +187,55 @@ def update_service_url(
         )
 
 
+@blockchain_options(contracts=[CONTRACT_DEPOSIT])
+@cli.command("withdraw")
+@click.option(
+    "--to", type=str, callback=validate_address, help="Target address for withdrawn tokens"
```

Comment:
Is this automatically required?

Reply:
No, it isn't. I'll change it. I first had it as an `argument`, which is always required. But then I noticed we don't use arguments anywhere and made it an option, again.